kaggle_environments/main.py

```diff
@@ -162,6 +219,10 @@ def action_handler(args):
             return {"error": "Already running a http server."}
         elif args.action == "act":
             return action_act(args)
+        elif args.action == "dispose":
+            return action_dispose(args)
+        elif args.action == "load":
```

Comment:
load now no longer requires an environment because the environment can be obtained from an input json file